codice/ddf-ui
# Resolve CVEs

Depends on:
- https://github.com/connexta/kanri/pull/43
- https://github.com/connexta/logviewer/pull/13

To test:
1. Run full build
2. `yarn start` in `ui-frontend/packages/admin`
3. Verify configurations can be created, modified and deleted
4. Verify the logs can be viewed and filtered
5. run `yarn audit --groups dependencies --level high`  in `ui-frontend/packages/admin` and verify no high or critical CVEs exist
